src/core/service.c

```diff
@@ -2116,7 +2106,7 @@ static void service_enter_dead(Service *s, ServiceResult f, bool allow_restart)
                  * can still introspect the counter. */
                 service_set_state(s, end_state);
 
-                service_set_debug_invocation(s, /* enable= */ false);
+                (void) unit_set_debug_invocation(UNIT(s), false);
```

Comment:
`/* enable = */` is dropped.

Reply:
I don't think it's actually necessary? E.g. we don't add such comments for fd_cloexec, because the name already clearly encodes the meaning/action here.

Reply:
yeah, i'd agree with @YHNdnzj. if the function name makes clear enough what the parameter is no need to add the extra config. hence, yes, if we have a setter call then i think it's pretty clear that first arg is the object to set the thing on, and 2nd arg is the value to set that thing to.